am-lib/src/main/resources/db/migration/V9__Create_Role.sql

```diff
@@ -0,0 +1,3 @@
+INSERT INTO roles VALUES ('case','ccd-user','Public');
```

Comment:
Please be consistent. Sometimes you use spaces between args, sometimes not. There should be always space between. Readability first.

Reply:
👍